src/ProfilesPage/ProfilePage.js

```diff
@@ -162,13 +176,19 @@ class ProfilePage extends Component {
 
   deleteUserButton() {
     const admin = sessionStorage.getItem("isAdmin");
-    return (
-      <a href="/message">
-        <button className="btn btn-danger" onClick={this.onDelete.bind(this)}>
-          Delete User
-        </button>
-      </a>
-    );
+    const superuser = this.props.userInfo.superuser;
+    
+    console.log(superuser);
+    {/*if superuser's value is false render deleleteUser button.*/}
+    if(superuser === false) {
```

Comment:
Could just use `if (!this.props.userInfo.superuser)` in place of creating the local `const`. And delete the console log and comment on 181-182.

Reply:
done